### PR TITLE
Add codecov to pipeline

### PR DESCRIPTION
**Pull request by trianglegirl**
To remind us to write more tests for jrNotes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # jrNotes 
+<!-- badges: start -->
 [![Build Status](https://api.travis-ci.org/jr-packages/jrNotes.png?branch=master)](https://travis-ci.org/jr-packages/)
+[![Codecov test coverage](https://codecov.io/gh/jr-packages/jrNotes/branch/master/graph/badge.svg)](https://codecov.io/gh/jr-packages/jrNotes?branch=master)
```
Comment (csgillespie) [inline README.md:4]: Do we need to add a env variable to travis?

File: .travis.yml [hunk 2]
```diff
@@ -26,4 +26,5 @@ script:
 
 after_success:
   - wget https://raw.githubusercontent.com/jr-packages/drat/master/deploy.sh
+  - Rscript -e 'covr::codecov()'
```
Comment (csgillespie) [inline .travis.yml:29]: Should only run on master?

**Review by csgillespie — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (trianglegirl) [reply README.md:4]: Nope. No need for public repos using Travis CI

File: .travis.yml (hunk 2 above)
Comment (trianglegirl) [reply .travis.yml:29]: I think sensible to only run on master

**Review by jwalton3141 — COMMENTED** [review]
Are there any files which we should be ignoring with a `.covrignore` file?

File: .travis.yml (hunk 2 above)
Comment (trianglegirl) [reply .travis.yml]: I think that resolves, although my travis.yml skills are not great.

**Comment by trianglegirl** [comment]
@jwalton3141 I can't think of any sensible files to add. But I can create a blank `.covrignore` to remind us and we can add later as we find?

**Review by csgillespie — APPROVED** [review]

**Review by jwalton3141 — APPROVED** [review]
yep. sounds sensible and all looks good to me.

**Pull request merged**